Broken Build due to "CMake 3.5.0 or higher is required.  You are running version 2.8.12.2."

Following README and running:

```
docker build \
  -t spring-boot:local \
  -f ./recipes/dockerfiles/spring-boot/Dockerfile \
  ./recipes/
```

currently causes:

```
oc
Loaded plugins: fastestmirror, ovl
Cleaning repos: base centos-sclo-rh centos-sclo-sclo extras updates
Cleaning up list of fastest mirrors
 ---> cb2fca2f77a0
Removing intermediate container f7f9eb11b36a
Step 5/21 : COPY ./context/nss_wrapper.install /tmp/install/
 ---> 6b6487e52478
Removing intermediate container 2ccfc0a0c7e8
Step 6/21 : RUN sudo chown user:user /tmp/install /tmp/install/* &&     /tmp/install/nss_wrapper.install &&     sudo rm -rf /tmp/install
 ---> Running in 77b076bcf45d

================================================================================
 Package             Arch        Version                     Repository    Size
================================================================================
Installing:
 cmake               x86_64      2.8.12.2-2.el7              base         7.1 M
 gcc                 x86_64      4.8.5-36.el7_6.1            updates       16 M
 make                x86_64      1:3.82-23.el7               base         420 k
Installing for dependencies:
 cpp                 x86_64      4.8.5-36.el7_6.1            updates      5.9 M
 glibc-devel         x86_64      2.17-260.el7_6.3            updates      1.1 M
 glibc-headers       x86_64      2.17-260.el7_6.3            updates      683 k
 kernel-headers      x86_64      3.10.0-957.10.1.el7         updates      8.0 M
 libarchive          x86_64      3.1.2-10.el7_2              base         318 k
 libmpc              x86_64      1.0.1-3.el7                 base          51 k
 lzo                 x86_64      2.06-8.el7                  base          59 k
 mpfr                x86_64      3.1.1-4.el7                 base         203 k

Transaction Summary
================================================================================
Install  3 Packages (+8 Dependent packages)

Total download size: 40 M
Installed size: 89 M

Installed:
  cmake.x86_64 0:2.8.12.2-2.el7          gcc.x86_64 0:4.8.5-36.el7_6.1         
  make.x86_64 1:3.82-23.el7             

Dependency Installed:
  cpp.x86_64 0:4.8.5-36.el7_6.1                                                 
  glibc-devel.x86_64 0:2.17-260.el7_6.3                                         
  glibc-headers.x86_64 0:2.17-260.el7_6.3                                       
  kernel-headers.x86_64 0:3.10.0-957.10.1.el7                                   
  libarchive.x86_64 0:3.1.2-10.el7_2                                            
  libmpc.x86_64 0:1.0.1-3.el7                                                   
  lzo.x86_64 0:2.06-8.el7                                                       
  mpfr.x86_64 0:3.1.1-4.el7                                                     

Cloning into 'nss_wrapper'...
CMake Error at CMakeLists.txt:2 (cmake_minimum_required):
  CMake 3.5.0 or higher is required.  You are running version 2.8.12.2


-- Configuring incomplete, errors occurred!
The command '/bin/sh -c sudo chown user:user /tmp/install /tmp/install/* &&     /tmp/install/nss_wrapper.install &&     sudo rm -rf /tmp/install' returned a non-zero code: 1
```

@tqvarnst this is exactly the same as what you locally hit as well, right?

@benoitf @dneary are you aware of this problem? Looks like this project as-is is broken...

It's blocking me for https://github.com/tqvarnst/quarkus-workshop/issues/2 so I'll see if I can figure something out (but no guarantees, both time and know how wise).

--- a/recipes/context/nss_wrapper.install
+++ b/recipes/context/nss_wrapper.install
@@ -1,5 +1,5 @@
 #!/bin/bash
-# Copyright (c) 2012-2017 Red Hat, Inc.
+# Copyright (c) 2012-2019 Red Hat, Inc.
 # All rights reserved. This program and the accompanying materials
 # are made available under the terms of the Eclipse Public License v1.0
 # which accompanies this distribution, and is available at
@@ -15,20 +15,8 @@ set -u
 set -e
 
 sudo yum update -y -d 1 \
+  && sudo yum install -y epel-release \
   && sudo yum install -y -d 1 \
-    cmake \
-    gcc \
-    make \
-  && cd /home/user/ \
-  && git clone git://git.samba.org/nss_wrapper.git \
-  && cd nss_wrapper \
-  && mkdir obj && cd obj \
-  && cmake -DCMAKE_INSTALL_PREFIX=/usr/local -DLIB_SUFFIX=64 .. \
-  && make && sudo make install \
-  && cd /home/user && rm -rf ./nss_wrapper \
-  && sudo yum remove -y \
-    cmake \
-    gcc \
-    make \
+    nss_wrapper \
   && sudo yum clean all \
   && sudo rm -rf /tmp/* /var/cache/yum

